src/pages/Admin/Admin_v2.tsx

```diff
@@ -24,34 +20,25 @@ import "./Admin.scss";
 
 dayjs.extend(advancedFormat);
 
+// @debt Rename component name and remove styled-components
```

Comment:
Can we repay it with this PR or is it OOS?

Does `AdminVenues` need to be wrapped anymore? Might be easier just to move `AdminRestricted` inside `AdminVenues` and delete `Admin_v2.tsx`